processing/src/main/java/io/druid/segment/serde/ComplexColumnSerializer.java

```diff
@@ -83,8 +84,11 @@ public long getSerializedSize()
   }
 
   @Override
-  public void writeToChannel(WritableByteChannel channel) throws IOException
+  public void writeToChannel(WritableByteChannel channel, FileSmoosher smoosher) throws IOException
   {
-    writer.writeToChannel(channel);
+    writer.writeToChannel(
+        channel,
+        null
+    ); //null for the FileSmoosher means that we default to "version 1" of GenericIndexed.
```

Comment:
Maybe better to add method `writeToChannelVersionOne()` that puts this null itself, than add such a comment